src/types.js

```diff
@@ -111,6 +111,7 @@ export type Message = {
   id: number,
   isOutbox: boolean,
   is_me_message: boolean,
+  last_edit_timestamp: number,
```

Comment:
This prop is optional.

Reply:
If the `EditMessage` type does not make sense, just make the prop optional with `?`.

Reply:
Hm, okay. I'll look into the code paths for editing messages.